crawlerr/server.py

```
@@ -7,8 +7,13 @@
 import asyncio
 import logging
 import sys
+import warnings
 from pathlib import Path
 from typing import Dict, Any
+import structlog
```

Ruff (0.12.2)

`structlog` imported but unused

Remove unused import: `structlog`

(F401)